### ISSUE TITLE
Right click menu hidden by task bar

### ISSUE THREAD

**Issue by Aesthetikx** [linuxmint/Cinnamon#91]
When right clicking on a file in nautilus, the right click menu appears (obviously).
Usually this is down and to the right of the mouse, if there is space available.  If the mouse is close to the bottom of the screen, the menu goes up and to the right from the mouse. (this is all as it should be).  

However, there is an inbetween state when you are just the right distance from the bottom of the screen where the bottom of the right click menu is covered by the bottom bar, as if only the space to the bottom of the screen is checked as opposed to the space to the top of the taskbar.

Hope that confusing explanation was atleast partially coherent.  Keep up the good work!

tl;dr, the bottom of the right click menu has the capability of being hidden by the taskbar.
Open up /etc and right click some folders in the middle of the window to reproduce.
